accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -805,9 +754,9 @@ class C {
 
 It's a compile-time error if:
 
-*   A `mixin class` declaration has a superclass other than `Object`. *The
-    declaration is limited to an `extends Object` clause or no `extends` clause,
-    and no `with` clauses. The class grammar prohibits `on` clauses.*
+*   A `mixin class` declaration has a superclass clause.
```

Comment:
What is a "superclass clause"? Do you mean "`extends` clause"?

Reply:
The entire
```ebnf
<superclass> ::= \EXTENDS{} <typeNotVoid> <mixins>?
    \alt <mixins>
```
clause, so named in the specification.